src/dataaccess/McodeV05SmartOnFhirDataSource.jsx

```diff
@@ -1,6 +1,5 @@
 import IDataSource from './IDataSource';
-import PatientRecord from '../patient/PatientRecord';
-import ObjectFactory from '../model/ObjectFactory';
+import processResources from './utils/fhir-entry-processor';
```

Comment:
One minor nit! Can we name processResources to processFhirResources since it specifically processes FHIR resources? Looks really good though! Thanks!